docs/validators/emergency-upgrades.md

```diff
@@ -0,0 +1,83 @@
+---
+order: 5
+title: Emergency Upgrades
+---
+
+## **Introduction**
+
+This article goes into the general flow of events for an emergency upgrade, pre-upgrade and during the upgrade for all validators.
+
+Emergency upgrades are an unfortunate necessity for all validators when running a PoS node.
+
+When a vunerability is discovered and responsibily disclosed, the vunerability will be reported to one of the channels described in the [Security.md](../../SECURITY.md).
+
+Once the vulnerability is patched and tested, the core team will reach out to all validators to update their nodes with the latest version.
+
+### **Cosmos Hub Upgrade Information**
+
+Depending on the severity of the vulnerability, the core teams and validators can take different actions.
+
+The core teams will try reach out **48 hours** or more before a scheduled upgrade, so that validators have time to inspect the changes and also adjust their internal planning accordingly.
+
+If the change is **non** state-breaking then the patch can be applied in a un-cordinated manner by validators asap.
+
+If the change is **state-breaking**, that will mean that the upgrade has to be co-ordinated for all validators at a certain block or halt-height.
+
+#### **Co-ordinated Upgrades**
+
+A co-ordinated upgrade will require updating your node configuration to the correct halt-height. The code teams will communicate the correct height for all validators via Email and [Discord](https://discord.com/channels/669268347736686612/798937713474142229).
+
+At the halt-height, you will swap out the old binaries for the new ones, while monitoring your node. Be sure to check-in on the [Cosmos Developers Discord](https://discord.gg/cosmosnetwork) on the [cosmos-hub-validators-verified](https://discord.com/channels/669268347736686612/798937713474142229), as notifications will be made pre/during/post upgrade on **this** channel.
```

Comment:
on the [cosmos-hub-validators-verified](https://discord.com/channels/669268347736686612/798937713474142229) in [Cosmos Developers Discord](https://discord.gg/cosmosnetwork)?

Reply:
> should be Cosmos Hub discord, and validators verified

This is the correct channel name:
<img width="278" alt="image" src="https://github.com/cosmos/gaia/assets/98309852/1b45e711-29dd-4d1a-9874-2b75943c744a">